### PR TITLE
Adding Even even More Features again

### PR DESCRIPTION
**Pull request by unbeatablesites**
Css fixes. Made the app extremely responsive!

### CODE REVIEW

**Comment by unbeatablesites** [comment]
Hope you like! 🥇

**Review by Surrealinsomniac — CHANGES_REQUESTED** [review]
It looks like you didn't pull the latest version before you made changes. I could be wrong but it looks like merging right now would overwrite changes I made to the html.

File: index.html [hunk 1]
```diff
@@ -5,7 +5,6 @@
 	<meta charset="utf-8" />
 	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
 	<link rel="stylesheet" href="assets/css/main.css" />
-	<link rel="icon" href="images/star-favicon.jpg">
```
Comment (Surrealinsomniac) [inline index.html:8]: this is something I added looks like you dont have.

File: index.html [hunk 2]
```diff
@@ -52,11 +51,10 @@ <h1>Five Star Moment</h1>
 		<div class="container">
 			<div class="content">
 				<header class="major">
-					<h2 id="section-label">See whats happening</h2><h1 id="events-label"></h1>
```
Comment (Surrealinsomniac) [inline index.html:55]: I added this

File: index.html [hunk 3]
```diff
@@ -81,7 +79,7 @@ <h2>Get in touch</h2>
 					<div class="col-12">
 						<ul class="actions special">
 							<li>
-								<input id="submit-button" type="submit" value="Send Message" class="primary" />
```
Comment (Surrealinsomniac) [inline index.html:84]: need this

File: index.html [hunk 4]
```diff
@@ -118,14 +116,14 @@ <h2>Get in touch</h2>
 			</ul>
 		</footer>
 	</section>
-	<script src="https://www.gstatic.com/firebasejs/5.2.0/firebase.js"></script>
```
Comment (Surrealinsomniac) [inline index.html:121]: correct firebase link

File: index.html [hunk 5]
```diff
@@ -118,14 +116,14 @@ <h2>Get in touch</h2>
...
 	</section>
-	<script src="https://www.gstatic.com/firebasejs/5.2.0/firebase.js"></script>
 			<script src="assets/js/jquery.min.js"></script>
 			<script src="assets/js/jquery.scrollex.min.js"></script>
 			<script src="assets/js/jquery.scrolly.min.js"></script>
 			<script src="assets/js/browser.min.js"></script>
 			<script src="assets/js/breakpoints.min.js"></script>
 			<script src="assets/js/util.js"></script>
 			<script src="assets/js/main.js"></script>
+			<script src="https://www.gstatic.com/firebasejs/5.2.0/firebase.js"></script>
```
Comment (Surrealinsomniac) [inline index.html:126]: incorrect firebase link

**Review by annievallone — APPROVED** [review]

**Review by Surrealinsomniac — APPROVED** [review]

**Pull request merged**